CHANGELOG.md

```diff
@@ -11,11 +11,24 @@ tag versions. The Bond compiler (`gbc`) and
 different versioning scheme, following the Haskell community's
 [package versioning policy](https://wiki.haskell.org/Package_versioning_policy).
 
+## Unreleased  ##
+* `gbc` & compiler library: TBD
+* IDL core version: TBD
+* IDL comm version: TBD
+* C++ version: TBD
+* C# NuGet version: TBD  (major bump needed)
+* C# Comm NuGet version: TBD
+
+### C# ###
+
+* **Breaking change** Reflection.IsBonded now recognizes custom IBonded
```

Comment:
For me this was on the border and I erred on the side of cautious. I will update to treat it as a bugfix.